tests/slurm-tests/slurm_test_llama_nemotron_super_49B_v1._5_evals/receipe.py

```diff
@@ -0,0 +1,463 @@
+    common_infer = [
+        "++inference.tokens_to_generate=65536",
+        "++inference.temperature=0.6",
+        "++inference.top_p=0.95",
+        "++skip_filled=True",
+        "++system_message=",
+    ]
+
+    # Math / Code / Science (Reasoning ON)
+    cmd_reasoning = [
+        "ns",
+        "eval",
+        "--cluster",
+        cluster,
+        "--model",
+        base_model,
+        "--server_type",
+        "vllm",
+        "--output_dir",
+        f"{workspace}/llama_nemotron_49b_1_5_reasoning_on",
+        "--benchmarks",
+        "gpqa:16,scicode:16,math-500:16,aime24:16,aime25:16",
+        "--server_gpus=8",
+        *common_infer,
+        "--run_after",
+        f"{expname_prefix}-download-ruler-data",
+        "--expname",
+        f"{expname_prefix}-math-code-science-on",
+    ]
+    subprocess.run(cmd_reasoning, check=True)
+
+    # MMLU needs two continous jobs run on A100 GPU
+    cmd_reasoning = [
+        "ns",
+        "eval",
+        "--cluster",
+        cluster,
+        "--model",
+        base_model,
+        "--server_type",
+        "vllm",
+        "--output_dir",
+        f"{workspace}/llama_nemotron_49b_1_5_reasoning_on",
+        "--benchmarks",
+        "mmlu-pro:16",
+        "--server_gpus=8",
+        "--dependent_jobs=1",
+        *common_infer,
+        "--run_after",
+        f"{expname_prefix}-download-ruler-data",
+        "--expname",
+        f"{expname_prefix}-math-code-science-on",
+    ]
+    subprocess.run(cmd_reasoning, check=True)
+
+    # LiveCodeBench (Reasoning ON)
+    cmd_livecode = [
+        "ns",
+        "eval",
+        "--cluster",
+        cluster,
+        "--model",
+        base_model,
+        "--server_type",
+        "vllm",
+        "--output_dir",
+        f"{workspace}/llama_nemotron_49b_1_5_reasoning_on",
+        "--benchmarks",
+        "livecodebench:16",
+        "--split",
+        "test_v5_2410_2502",
+        "--server_gpus=8",
+        *common_infer,
+        "--run_after",
+        f"{expname_prefix}-download-ruler-data",
+        "--expname",
+        f"{expname_prefix}-livecode-on",
+    ]
+    subprocess.run(cmd_livecode, check=True)
+
+    # HLE (Reasoning ON)
+    cmd_hle = [
+        "ns",
+        "eval",
+        "--cluster",
+        cluster,
+        "--model",
+        base_model,
+        "--server_type",
+        "vllm",
+        "--output_dir",
+        f"{workspace}/llama_nemotron_49b_1_5_reasoning_on",
+        "--benchmarks",
+        "hle:16",
+        "--server_gpus=8",
+        "--dependent_jobs=1",
+        "--judge_model",
+        f"{workspace}/Qwen2.5-32B-Instruct",
+        "--judge_server_type=vllm",
+        "--judge_server_gpus=8",
+        "--extra_judge_args=++inference.tokens_to_generate=4096 ",
+        *common_infer,
+        "--run_after",
+        f"{expname_prefix}-download-ruler-data",
+        "--expname",
+        f"{expname_prefix}-hle-on",
+    ]
+    subprocess.run(cmd_hle, check=True)
```

Comment:
you can use `shell=True` and lot of the logic can then be shared between all the evals.